src/pkg/bundle/common.go

```diff
@@ -104,6 +104,11 @@ func (b *Bundle) ValidateBundleResources(bundle *types.UDSBundle, spinner *messa
 
 	// validate access to packages as well as components referenced in the package
 	for idx, pkg := range bundle.Packages {
+		// if package path is set, make it relative to source directory
+		if pkg.Path != "" {
+			pkg.Path = fmt.Sprintf("%s/%s", b.cfg.CreateOpts.SourceDirectory, pkg.Path)
```

Comment:
```suggestion
			pkg.Path = filepath.Join(b.cfg.CreateOpts.SourceDirectory, pkg.Path)
```
Recommend using the library call instead of building out the path manually (this makes all OSs happy)